tests/test_reports.py

```diff
@@ -382,3 +382,33 @@ def test_expression_parsing(expr, expected_filters):
     assert expr_to_filters(expr) == Filters(
         op="OR", filters=[Filters(op="AND", filters=expected_filters)]
     )
+
+
+def test_layout_config():
+    DEFAULT_LAYOUT = {
+        "x": wr.Layout.x,
+        "y": wr.Layout.y,
+        "w": wr.Layout.w,
+        "h": wr.Layout.h,
+    }
+    CUSTOM_USER_DEFINED_LAYOUT = {"x": 0, "y": 0, "w": 24, "h": 24}
+
+    p0 = wr.interface.WeavePanelSummaryTable(table_name="test")
```

Comment:
nit: I think you can just do `wr.WeavePanelSummaryTable`.  If you cant we should prob fix that :)